### DISCUSSION TITLE
Gallery view : too long (too vertical)

### DISCUSSION THREAD

**Discussion by Dont-ask-for-it** [nocodb/nocodb#4344] [Features]
Need columns

**Comment by dstala** [comment]
Could you elaborate more on the requirement. Is it just change in card dimensions (height) in Gallery view? You can shrink it by reducing number of selected fields to be displayed with in gallery card. Columns with in a card is not in our current road map.

**Comment by Dont-ask-for-it** [comment]
1, Without going 2 columns, the css could be changed to offer more compact placement. 
2. The photo could be implemented on very top (no margin) and 100% for a better look
3. Tags / multiselect fields : the tag could be on the same line (no line break between each tags)

Here's some examples of this with Airtable and Baserow:

![image](https://user-images.githubusercontent.com/5061749/201138879-dfff0fec-23d5-4920-843f-5e1c8183ec19.png)
![image](https://user-images.githubusercontent.com/5061749/201139021-1035192c-8c31-4b21-9bbb-63c370ec5629.png)

**Comment by Dont-ask-for-it** [comment]
Here's my view to illuistrate the images placement and multiselects on multiples lines :

![image](https://user-images.githubusercontent.com/5061749/201141468-f392c28b-4c4f-4550-9438-859294916c22.png)
